internal/v2/remote_signer/remote_signer.go

```diff
@@ -2,13 +2,77 @@
 package remotesigner
 
 import (
+	"io"
 	"crypto"
-	"crypto/tls"
+	"crypto/x509"
+	s2av2pb "github.com/google/s2a-go/internal/proto/v2/s2a_go_proto"
+	commonpbv1 "github.com/google/s2a-go/internal/proto/common_go_proto"
 )
 
+type RemoteSigner struct {
+	leafCert *x509.Certificate
+	cstream s2av2pb.S2AService_SetUpSessionClient
+	hostname string
```

Comment:
What hostname is this? Should we maybe be storing the local identity instead?

If so, I think it should be:
```
cpb "github.com/google/s2a-go/internal/proto/common_go_proto"

localIdentity *cpb.Identity
```

Reply:
This is the identity we're tying to SessionReq in the Sign API. That's a good idea, I'll change it to this, thanks!